packages/is-shallow-equal/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Note for future reference in extracting rules: We should consider having separate rulesets for ES5 vs. ES2015+ .

Looks like this is already built-in to `eslint-plugin-wordpress`:

https://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress#available-rulesets

(Confused why there's both a config and a plugin which includes rulesets)

Reply:
Agreed, it would be much easier to put the following:

```json
{
    root: true,
    extends: "something"
}
```

Reply:
Yeah, I do this in my own projects, where (like with this one) I sometimes intentionally author in ES5.

https://github.com/aduth/memize/blob/4c0c9f9/.eslintrc.json#L3
https://github.com/aduth/equivalent-key-map/blob/e80140c/.eslintrc.json#L3